src/client/components/forms/userCollection.js

```diff
@@ -262,21 +262,25 @@ class UserCollectionForm extends React.Component {
 							<div className={errorAlertClass}>
 								<Alert bsStyle="danger">Error: {errorText}</Alert>
 							</div>
-							<div className="text-center">
-								<Button
-									bsStyle="primary"
-									type="button"
-									onClick={this.handleAddCollaborator}
-								>
-									<FontAwesomeIcon icon="plus"/>
-									&nbsp;Add another collaborator
-								</Button>
-								<Button
-									bsStyle="success"
-									type="submit"
-								>
-									<FontAwesomeIcon icon="save"/>&nbsp;{submitLabel}
-								</Button>
+							<div className="row">
+								<div className="col-sm-6 margin-top-d5">
+									<Button
```

Comment:
```suggestion
									<Button
										block
```

This makes the button take up all the available space, which I think looks quite nice:

Before:
<img width="418" alt="Capture d’écran 2020-11-10 à 15 51 10" src="https://user-images.githubusercontent.com/6179856/98689838-9a791a00-236c-11eb-82a9-a56f29ee8edf.png">


After:
<img width="367" alt="Capture d’écran 2020-11-10 à 15 43 48" src="https://user-images.githubusercontent.com/6179856/98689728-7ddce200-236c-11eb-84d7-219de18231a3.png">